ivy/functional/frontends/jax/numpy/logic.py

```diff
@@ -284,3 +284,31 @@ def packbits(x, /, *, axis=None, bitorder="big"):
     bits = ivy.expand_dims(bits, axis=tuple(range(x.ndim - 1)))
     packed = (x << bits).sum(axis=-1).astype("uint8")
     return ivy.swapaxes(packed, axis, -1)
+
+
+@with_supported_dtypes(
```

Comment:
I would like to share a quick consideration: I'm curious about the potential rationale behind the differentiation in supported data types for the `fromfunction` and `setxor1d` functions compared to other functions that appear to accommodate a wider range of data types. Is there a particular justification for this distinction that I might be overlooking? Your insights on this matter would be greatly appreciated.

Reply:
@abdul, you are absolutly correct, the supported wrapper is not needed anymore after i modified the func, thanks.
will push the changes it soon.

and will test setxor1d again as well.